src/supertux/game_session.hpp

```diff
@@ -88,6 +95,11 @@ class GameSession final : public Screen,
   /** ends the current level */
   void finish(bool win = true);
   void respawn(const std::string& sectorname, const std::string& spawnpointname);
+  void respawn_with_fade(const std::string& sectorname,
+                         const std::string& spawnpointname,
+                         const FadeType fade_type,
+                         const Vector fade_point,
```

Comment:
I wonder if passing by reference here is preferable.

Reply:
I think so.

Reply:
Good idea.